perception/realsense_driver/python/realsense_wrapper/realsense_d435.py

```diff
@@ -51,12 +56,25 @@ def get_intrinsics(self):
     def get_num_cameras(self):
         return len(self.device_ls)
 
-    def get_images(self):
+    def get_rgbd(self):
+        """Returns a numpy array of [n_cams, height, width, RGBD]"""
         framesets = self._get_frames()
-        imgs = []
-        for frameset in framesets:
-            frame = frameset.get_color_frame()
-            img = np.asanyarray(frame.get_data())
-            imgs.append(img)
+        num_cams = self.get_num_cameras()
+
+        rgbd = np.zeros([num_cams, self.height, self.width, 4], dtype=np.uint16)
```

Comment:
minor - consider using np.ndarray to save the time zeroing the array as the values are going to be overwritten anyways.

Reply:
`np.empty` right? will do in a future PR